packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -107,6 +107,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     toolbar?: boolean;
     toolbarActionProp?: boolean;
     rowActionProp?: boolean;
+    selectionProps?: "multiSelect" | "singleSelect";
```

Comment:
Naming: Maybe just `selection`, as it does more than adding support for props?

Reply:
in detail it's not that much more than adding props. forwarding two props, setting two or three to a specific value. The AssignProductsGrid containing selection-state-management is currently not generated but handmade. Generating the AssignProductsGrid would probably even be a complete different generator-config-type.